File: lab/javascript/POO/retangulo.js
class Retangulo{
    constructor(largura, altura){
        this.largura = largura;
        this.altura = altura;
    }

    repr(){
        return `Retangulo(largura=${this.largura}, altura=${this.altura})`;
    }

    area(){
        return this.largura * this.altura;
    }
}


ret = new Retangulo(4, 5);
console.log(ret.repr());
console.log(`Área = ${ret.area()}`);

File: lab/python/POO/retangulo.py
class Retangulo:
    def __init__(self, largura, altura) -> None:
        self.largura = largura
        self.altura = altura

    def __repr__(self) -> str:
        classe = self.__class__.__name__
        return f"{classe}(largura={self.largura}, altura={self.altura})"

    def area(self):
        return self.largura * self.altura


ret = Retangulo(4, 5)
print(ret)
print(f"Área = {ret.area()}")
